go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/cyphar/filepath-securejoin v0.2.2 // indirect
 	github.com/deislabs/smi-sdk-go v0.0.0-20190610232231-f281e2121a16
 	github.com/dgrijalva/jwt-go v3.1.0+incompatible // indirect
+	github.com/docker/docker v0.7.3-0.20190327010347-be7ac8be2ae0
```

Comment:
@KIVagant could you please remove this and run `bin/update-go-deps-shas` on more time?

Reply:
I'm trying right now but I barely understand what I'm doing

Reply:
You introduced a dependency on the `docker` lib in your earlier commits, which resulted in this line being added. All the `Dockerfile` files contain a checksum of the `go.mod` file, so every time it changes the `Dockerfile` files need to be updated, which is done through `bin/update-go-deps-shas`. That dependency is no longer needed, so it's just a matter of manually removing that line, and then running `bin/update-go-deps-shas`.

Reply:
Looks like now I've got it, thank you! :) Removed the line and the sha is the same now as in the master.